package.json

```diff
@@ -15,13 +15,15 @@
     "start": "npm start --prefix packages/template-retail-react-app",
     "pretest": "npm run lint",
     "test:e2e": "npx playwright test",
+    "test:e2e:update-snapshots": "npx playwright test --update-snapshots",
```

Comment:
Do we need to run this each time ?
If yes then should we be adding a step in the CI  workflow to run this before the tests run ?

Reply:
We don't need to run it each, only when we want to update snapshots